notebooks/notebooks/iclr2022/elo-vs-visits.ipynb

```diff
@@ -128,7 +128,7 @@
    "outputs": [
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #21.            "elo": float(elo_line.split(":")[1].split("+/-")[0]) + 635.812,
</pre><p>Where is the 635.812 number coming from? I guess it's setting cp505-v1 as the zero point -- can we do that automatically?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/HumanCompatibleAI/KataGoVisualizer/pull/24/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>